dbt_subprojects/daily_spellbook/models/_projects/lifi/lifi_transfers.sql

```diff
@@ -0,0 +1,49 @@
+{{
+    config(
+        schema = 'lifi',
+        alias = 'transfers',
+        materialized = 'view',
+        contributor = 'lequangphu'
```

Comment:
```suggestion
        materialized = 'view',
        post_hook='{{ expose_spells(\'[
                                        "arbitrum"
                                        , "avalanche_c"
                                        , "bnb"
                                        , "ethereum"
                                        , "fantom"
                                        , "gnosis"
                                        , "zksync"
                                    ]\',
                                    "project",
                                    "lifi",
                                    \'["lequangphu"]\') }}'
```

Reply:
if you want to display as a contributor, you need to have within a post-hook which will display this final crosschain view in the data explorer and then your name at the bottom

Reply:
please revisit this note here :)

Reply:
@lequangphu i can't merge until this one is resolved. we don't run any actions in the model config in CI env, only once merged to prod. we should ensure we have the `post_hook` setup properly like my proposal above 🙏

Reply:
oh, i miss it, will do.